src/NuGet.Clients/NuGet.PackageManagement.UI/UserInterfaceService/NuGetSettings.cs

```diff
@@ -1,11 +1,11 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System;
 using System.Collections.Generic;
 using NuGet.PackageManagement.UI;
```

Comment:
Now that it lives in NuGet.PackageManagement.UI, this using statement is redundant.